docs/recipes.md

```diff
@@ -70,17 +70,61 @@ jrnlimport () {
 
 ### Using templates
 
-Say you always want to use the same template for creating new entries.
-If you have an [external editor](../advanced) set up, you can use this:
+!!! note
+    Templates require an [external editor](../advanced) be configured. 
+
+A template is a code snippet that makes it easier to enter use repeated text 
+each time a new journal entry is started. There are two ways you can utilize
+templates in your entries.  
+
+#### 1. Command line arguments
+
+If you had a `template.txt` file with the following contents:
 
 ```sh
-jrnl < my_template.txt
-jrnl -1 --edit
+My Personal Journal
+Title: 
+
+Body:
 ```
 
-Another nice solution that allows you to define individual prompts comes
-from [Jacobo de
-Vera](https://github.com/maebert/jrnl/issues/194#issuecomment-47402869):
+The `template.txt` file could be used to create a new entry with these 
+command line arguements:
+
+```sh
+jrnl < template.txt     # Imports template.txt as the most recent entry
+jrnl -1 --edit          # Opens the most recent entry in the editor 
+```
+
+#### 2. Include the template file in `jrnl.yaml`
+
+A more efficient way to work with a template file is to declare the file
+in your config file by changing the `template` setting from `false` to the
+template file's path in double quotes:
+
+```sh
+...
+tagsymbols: "@"
+template: "/path/to/template.txt"
+timeformat: '%Y-%m-%d %H:%M'
+version: "2.1.1"
```

Comment:
What do you think about just showing the "template" field here? I'm of two minds - it's nice to show the neighboring fields for context, but also, I'm afraid that leaving the version field in here may be confusing in the long run.